### PR TITLE
{lib}[dummy/dummy] daal v2017.1.132

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/daal/daal-2017.1.132.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ Free and open source community-supported versions are available, as 
+ well as paid versions that include premium support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_%(name)s_%(version)s.tgz']
+
+dontcreateinstalldir = 'True'
+
+license_file = '/opt/intel/licenses/license.lic'
```
Comment (boegel) [inline easybuild/easyconfigs/d/daal/daal-2017.1.132.eb]: please use `HOME + '/licenses/intel/license.lic'`

changing this to something else is considered to be site-specific

File: easybuild/easyconfigs/d/daal/daal-2017.1.132.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+easyblock = "IntelBase"
```
Comment (boegel) [inline easybuild/easyconfigs/d/daal/daal-2017.1.132.eb]: style nitpicking: please include empty line below this line

**Comment by boegel** [comment]
@shahzebsiddiqui anything missing in the generated module file here, e.g. updates to `$PATH`, `$LD_LIBRARY_PATH`, etc.?

**Comment by shahzebsiddiqui** [comment]
I think we need to fix the modulefile. I did a difference before and after sourcing daalvar.sh 

The default PATH set in module file is setting to a directory where compilervars.sh which is the same when loading intel parallel studio. That script tries to source all the other components which I don't think we want in a module file. 
 
[daal_env_differences.txt](https://github.com/hpcugent/easybuild-easyconfigs/files/784312/daal_env_differences.txt)

File: easybuild/easyconfigs/d/daal/daal-2017.1.132.eb (hunk 2 above)
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/d/daal/daal-2017.1.132.eb]: Do you want to resubmit this or did you make the change?

**Comment by shahzebsiddiqui** [comment]
See PR 4825, this should be closed.

**Pull request closed without merging**